### PR TITLE
Add SCIM fields tenant apis

### PR DESCRIPTION
**Pull request by mooreds**
Adds missing SCIM fields to tenant API doc

### CODE REVIEW

File: site/docs/v1/tech/apis/_tenant-request-body.adoc [hunk 1]
```diff
@@ -465,6 +465,57 @@ The length of time in seconds this JWT is valid from the time it was issued. Val
...
+[field]#tenant.lambdaConfiguration.scimGroupRequestConverterId# [type]#[UUID]# [optional]#Optional# [since]#Available since 1.36.0#::
+The Id of a SCIM Group Request Lambda that will be used to convert the SCIM Group request to a FusionAuth Group.
++
+:enterprise_feature: SCIM
+include::../shared/_enterprise-edition-blurb-api.adoc[]
++
+Required when [field]#tenant.scimServerConfiguration.enabled# is `true`.
+
+[field]#tenant.lambdaConfiguration.scimGroupResponseConverterId# [type]#[UUID]# [optional]#Optional# [since]#Available since 1.36.0#::
+The Id of a SCIM GroupResponse Lambda that will be used to convert a FusionAuth Group to a SCIM Server response.
```
Comment (robfusion) [inline site/docs/v1/tech/apis/_tenant-request-body.adoc:493]: GroupResponse missing a space there I think.

File: site/docs/v1/tech/apis/_tenant-request-body.adoc [hunk 2]
```diff
@@ -751,6 +802,34 @@ include::../shared/_premium-edition-blurb-api.adoc[]
...
+Required when [field]#tenant.scimServerConfiguration.enabled# is `true`.
+
+[field]#tenant.scimServerConfiguration.enabled# [type]#[Boolean]# [optional]#Optional# [since]#Available since 1.36.0#::
+Whether or not this tenant has the SCIM endpoints enabled.
++
+:enterprise_feature: SCIM
+include::../shared/_enterprise-edition-blurb-api.adoc[]
+
+[field]#tenant.scimServerConfiguration.schemas# [type]#[Map]# [optional]#Optional# [since]#Available since 1.36.0#::
+JSON formatted as a SCIM Schemas endpoint response. Because the SCIM lambdas may modify the JSON response, ensure the Schema's response matches that generated by the response lambdas.
```
Comment (robfusion) [inline site/docs/v1/tech/apis/_tenant-request-body.adoc:820]: Maybe add a link to the SCIM Schemas schema definition here.

https://datatracker.ietf.org/doc/html/rfc7643#section-7

File: site/docs/v1/tech/apis/_tenant-response-body-base.adoc [hunk 3]
```diff
@@ -488,6 +506,18 @@ The duration for the number of times a user can request a two-factor code by ema
...
 A list of unique domains that are not allowed to register when self service is enabled.
 
+[field]#{base_field_name}.scimServerConfiguration.clientEntityTypeId# [type]#[UUID]# [since]#Available since 1.36.0#::
+The Entity Type that will be used to represent SCIM Clients for this tenant.
+
+[field]#{base_field_name}.scimServerConfiguration.enabled# [type]#[Boolean]# [since]#Available since 1.36.0#::
+Whether or not this tenant has the SCIM endpoints enabled.
+
+[field]#{base_field_name}.scimServerConfiguration.schemas# [type]#[Map]# [since]#Available since 1.36.0#::
+JSON formatted as a SCIM Schemas endpoint response. Because the SCIM lambdas may modify the JSON response, ensure the Schema's response matches that generated by the response lambdas.
```
Comment (robfusion) [inline site/docs/v1/tech/apis/_tenant-response-body-base.adoc:516]: Same as above. A link to the SCIM Schemas schema might be helpful and maybe a mention of the default value.

Comment (mooreds) [reply site/docs/v1/tech/apis/_tenant-response-body-base.adoc:516]: Sounds good. I think that the link is helpful, but the default value belongs in the request, not the response.

**Comment by mooreds** [comment]
Per Rob, looks good.

**Pull request merged**